src/xtopcom/xblockmaker/src/xfulltable_statistics.cpp

```diff
@@ -160,10 +105,8 @@ xstatistics_data_t tableblock_statistics(std::vector<xobject_ptr_t<data::xblock_
             xassert(!blks[i]->get_cert()->get_audit_signature().empty());
         }
 
-        uint32_t vote_num = cal_vote_num(blks[i], is_auditor);
-        tableblock_statistics_handle(leader_xip, txs_count, 0, data);
-        process_vote_info(blks[i], data);
-
+        calc_workload_data(leader_xip, txs_count, data);
```

Comment:
to every block, there will be two multisig string: auditor, validator;  after this modify,  you only process leader's multisig string(leader group's vote info)